libcontainer/landlock/landlock_unsupported.go

```diff
@@ -0,0 +1,19 @@
+// +build !linux
+
+package landlock
+
+import (
+	"errors"
+
+	"github.com/opencontainers/runc/libcontainer/configs"
+)
+
+var ErrLandlockNotSupported = errors.New("land: config provided but Landlock not supported")
+
+// InitLandlock does nothing because Landlock is not supported.
+func InitSLandlock(config *configs.Landlock) error {
+	if config != nil {
+		return ErrLandlockNotSupported
+	}
+	return nil
+}
```

Comment:
runc is linux-only, I guess you can drop this entirely (as well as `_linux` suffix from all the other files).